matscipy/dislocation.py

```diff
@@ -629,8 +635,11 @@ def make_screw_cyl_kink(alat, C11, C12, C44,
     b = np.sqrt(3.0) * alat / 2.0
     cent_x = np.sqrt(6.0) * alat / 3.0
 
-    disloc_ini, disloc_fin, bulk_ini = make_barrier_configurations((alat, C11, C12, C44),
-                                                                    cylinder_r=cylinder_r, **kwargs)
+    disloc_ini, \
```

Comment:
Better style to avoid need for `\` line continuation by enclosing values within partentheses.

Reply:
Ah yes, I should try that. I did it just because of the flake8 linter complains about the lengths of the line. I saw in the guidelines that we do not care much about it. So I see what looks more readable. I agree that `\` are ugly